src/test/java/io/github/wistefan/mapping/desc/pojos/subscription/MySubscriptionPojo.java

```diff
@@ -0,0 +1,46 @@
+package io.github.wistefan.mapping.desc.pojos.subscription;
+
+import io.github.wistefan.mapping.annotations.MappingEnabled;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.net.URI;
+import java.time.Instant;
+import java.util.List;
+import java.util.Set;
+
+@EqualsAndHashCode
+@ToString
+@Getter
+@MappingEnabled(subscriptionType = "Subscription")
```

Comment:
Is the subscriptionType field required to be the same that we have in the type field? If so then we could use the const here too or?

Reply:
Yes, you're right